### DISCUSSION TITLE
Request to manually edit dependencies

### DISCUSSION THREAD

**Discussion by ghost** [marimo-team/marimo#1790] [General]
### Describe the bug

Hello! I was wondering if there could be a way to manually edit the trees in the dependencies tab? I like how Marimo lists dependencies based on variables present in previous cells, but I'm having trouble running cells that have functions and edit a dataframe, but then those functions aren't used in the next cells. I have cells later on in my code that call the dataframe, but I need them to run after I have edited the dataframe with the functions. Could there be a function to edit the trees to allow this to happen?

### Environment

0.7.3

**Comment by mscolnick** [comment]
Moving to a discussion:

The dependency tree is all inferred based of the cells that define a variable/function and the cells that reference those. You can learn more here: https://docs.marimo.io/guides/reactivity.html
